### ISSUE TITLE
Incorrect channels

### ISSUE THREAD

**Issue by Wnt2Fly** [tonywagner/EPlusTV#80]
EplusTV is up and running just fine, and showing output correctly but all data is on the wrong channels. For example eplus says channel 11 for a specific game but its really on 39. Nothing is matching up.

**Comment by Wnt2Fly** [comment]
ESPN+ EPG channel mapping still not working correctly. Today i checked my IPTV provider and they showed a specific game on ESPN+ 23, which was correct. Guide showed it on another channel in the 30s..
I have properly mapped all my channels to the right feeds.

thoughts?

**Comment by m0ngr31** [comment]
What does your docker run command look like?

**Comment by Wnt2Fly** [comment]
Its out of the box default, haven't changed anything.

**Comment by m0ngr31** [comment]
I mean how are you running the container?

**Comment by Wnt2Fly** [comment]
Sorry, really new to docker and such. I've got a Synology NAS drive w/the container system installed there. Its running fine, the mappings of channel #s are just off.. I'm not sure if its my IPTV provider moving things around or what...

I'm happy to run whatever command you'd like me to, sorry just so new to it.

**Comment by m0ngr31** [comment]
How is the container setup in the Synology software? Like what variables are you using and stuff?

What IPTV provider are you using? Wondering if the unescaped characters are breaking imports?

**Comment by Wnt2Fly** [comment]
Oh sorry, yah everything is default, so i didn't set an env vars at all.. truly 100% out of the box install.
Happy to run debug code if you want me to test anything.  possible if you could output the current channel # and program you are getting from ESPN, and then we could compare that to the XML output..

**Comment by m0ngr31** [comment]
Just so I'm clear, you are setting no environment variables on the container? So you're only using ESPN+? I'm not sure what IPTV you're using has any bearing, the only thing that matters is the client.

**Comment by Wnt2Fly** [comment]
Correct, no env set other than the path. I mapped your channel 1 to my provider channel 1, 2 to 2, 3 to 3.. etc. in Xteve.. the EPG only matches up now and then for some shows and not for others.

**Comment by burntoc** [comment]
I've seen this or similar behavior for months. I think whenever I have to reboot the container the channel numbers can switch between my 8 or so stations (I don't use ESPN+, MLB, etc), and if I do it pretty close to back to back (for say, server maintenance) the guide channels can be off.  I'm guessing the channel numbers switching is the primary cause, and maybe the guide data isn't getting refreshed with the new numbers.  Can we dedicate channel numbers to each channel?
